USAGE.md

```diff
@@ -210,37 +239,38 @@ $ tw launch https://github.com/nf-core/rnaseq --params-file=./custom_rnaseq_para
     https://tower.nf/user/abhinav/watch/2XDXxX0vCX8xhx
 ```
 
-> **NOTE**: Platform CLI users are bound to the same user permissions that apply in the platform UI. Launch users can launch pre-configured pipelines in the workspaces they have access to, but they cannot add or run new pipelines.
-
-In the above command:
+- Pipeline parameters are defined within the `custom_rnaseq_params.yaml` file.
+- Other parameters such as `--profile` and `--revision` can also be specified.
+- A non-primary compute environment can be used to launch the pipeline. Omit `--compute-env` to launch with the workspace default compute environment.
 
-- Pipeline level parameters are defined within the `custom_rnaseq_params.yaml` file
-- Other parameters such as `--profile` and `--revision` can also be specified
-- A non-primary Compute Environment has been used to launch the pipeline
+> **Note**: CLI users are bound to the same user permissions that apply in the Platform UI. Launch users can launch pre-configured pipelines in the workspaces they have access to, but they cannot add or run new pipelines.
 
 ## Workspaces
 
-Workspaces provide the context in which a user operates, i.e. to launch workflow executions, define the available resources and to manage who can access/operate on those resources. Workspaces are composed of Pipelines, Runs, Actions, Datasets, Compute Environments and Credentials. Access permissions are controlled through Participants, Collaborators, and Teams.
+Run `tw workspaces -h` to view supported workspace operations.
+Run `tw workspaces add -h` to view the required and optional fields for adding your workspace.
+
+Workspaces provide the context in which a user launches workflow executions, defines the available resources, and manages who can access those resources. Workspaces contain pipelines, runs, actions, datasets, compute environments, and credentials. Access permissions are controlled with participants, collaborators, and teams.
```

Comment:
Is "Workspaces contain pipelines, runs, actions, datasets, compute environments, and credentials." a comprehensive list? If so, we should add "secrets" to the list